### DISCUSSION TITLE
Multiple Hosts

### DISCUSSION THREAD

**Discussion by holocronology** [louislam/dockge#12] [Feature Request]
Add ability to manage multiple docker hosts/environments from one Dockge instance.

**Comment by louislam** [comment]
Related pull request: https://github.com/louislam/dockge/pull/3

**Comment by iYoungblood** [comment]
+1 to this request. I have many hosts in my home lab and running from one dockge would be ideal. Having to install dockge on every host doesnt scale well.

**Comment by louislam** [comment]
1.4.0-beta.0 has been released. It supports multiple Dockge instances now.

If you would like to test it, feel free to switch to the beta version.

```yaml
...
services:
  dockge:
    image: louislam/dockge:beta
...
```

Read more:
https://github.com/louislam/dockge/releases/tag/1.4.0-beta.0

**Comment by cfpwastaken** [comment]
My setup has the stacks directory mounted on both machines. Thus both machines see the same stacks, but only one runs a specific stack. In Dockge they show up twice now, running and not running. Will a setup like this get supported later?
